docs/faq/helium-network.mdx

```diff
@@ -91,17 +87,21 @@ Hotspot owners earn HNT depending on how frequently their devices do two things:
 ### What if I'm in a rural area, or the only Hotspot in my area? Can I still earn rewards?
 
 It's best to have at least one or two other Hotspots within radio range of you so that you can
-participate in Proof of Coverage Challenges to earn HNT rewards at a higher rate. If you are the
-only Hotspot in your area, you can still earn HNT if a device uses your Hotspots and by initiating
-the Challenges. HNT is distributed based on these economics. Read more about token economics
-[here](https://www.helium.com/token).
+participate in Proof of Coverage Challenges to earn IOT rewards at a higher rate. If you are the
+only Hotspot in your area, you will earn IOT if a device uses your Hotspots connection to transfer
+sensor data back to the sensor users. IOT is distributed based on these economics. Read more about 
+token economics [here](https://www.helium.com/token).
 
-### Who governs the Helium Network and what is the Helium Foundation?
+### Can I connect my phone or computer to Hotspots?
 
-The Helium Foundation is a global not-for-profit dedicated to the proliferation of secure,
-cost-effective connectivity and it manages and maintains the open technologies that support the
-Helium Network. You can read more about governance from Helium Foundation
-[here](https://helium.foundation).
+Yes. As of early 2022 the community has started to roll out 5G-CBRS hotspots in the USA
+[4G/LTE and 5G coverage on the Helium blockchain](https://www.helium.com/5G).
+
+### Does Helium partner with Internet providers?
+
+Yes. In fact, Helium recently partnered with DISH, a nationwide provider of internet connectivity,
```

Comment:
2021 isn't exactly recent. 
Is this a common question? If we feel like we need to keep it, t-mobile might be more relevant